index.html

```diff
@@ -0,0 +1,64 @@
+<!DOCTYPE html>
+<html>
+<head>
```

Comment:
Use consistent spacing between head tags. Also, close meta tags properly and consistently.